server/routes/referrals/referralsController.test.ts

```diff
@@ -143,7 +143,7 @@ describe('POST /intervention/:id/refer', () => {
         .send({ 'service-user-crn': 'X123456' })
         .expect(400)
         .expect(res => {
-          expect(res.text).toContain('CRN not found in nDelius')
+          expect(res.text).toContain('CRN not found in nDelius (CRNs are case sensitive)')
```

Comment:
Doesn't seem like this belongs here (seems related to your previous comment)

Reply:
thanks - i forgot to remove it